Package.swift

```diff
@@ -169,7 +169,7 @@ let package = Package(
     cxxLanguageStandard: .gnucxx17
 )
 
-if Context.environment["MLX_SWIFT_BUILD_DOC"] == "1" {
+if Context.environment["MLX_SWIFT_BUILD_DOC"] == "1" || Context.environment["SPI_GENERATE_DOCS"] == "1" {
```

Comment:
This line fails the swift-format check.  Can you run:

```
pre-commit run --all-files
```

It wants to make this change:

```
-if Context.environment["MLX_SWIFT_BUILD_DOC"] == "1" {
+if Context.environment["MLX_SWIFT_BUILD_DOC"] == "1"
+    || Context.environment["SPI_GENERATE_DOCS"] == "1"
+{
```

Thank you!